compiler-rs/clients_schema_to_openapi/src/paths.rs

```diff
@@ -311,6 +313,25 @@ fn get_path_parameters(template: &str) -> Vec<&str> {
     result
 }
 
+fn split_summary_desc(desc: String) -> SplitDesc{
+    let mut parts = desc.split(['.','\n',':']);
```

Comment:
I don't think we should split on `:` as the description can become a bit weird.

We only have one occurrence of this: "Returns information about indices: number of primaries and replicas, document counts, disk size, ...". The entire sentence is ok IMHO to use as the summary.